api/proto/config/devconfig.proto

```diff
@@ -136,6 +136,13 @@ message EdgeDevConfig {
   // power on capability; power off can be done locally using the Local Profile
   // Server API.
   DeviceOpsCmd shutdown = 34;
+
+  string device_name = 35;
+  string device_id = 36;
```

Comment:
That is already in the message; I think it is called "id" at the top of the EdgeDevConfig.